android/src/app/organicmaps/editor/AdvancedTimetableFragment.java

```diff
@@ -137,7 +143,21 @@ private static void setTextChangedListener(@Nullable EditText input,
       public void beforeTextChanged(CharSequence s, int start, int count, int after) {}
 
       @Override
-      public void onTextChanged(CharSequence s, int start, int before, int count) {}
+      public void onTextChanged(CharSequence s, int start, int before, int count) {
```

Comment:
Please follow our [code formatting rules](https://github.com/organicmaps/organicmaps/blob/master/docs/JAVA_STYLE.md).

Reply:
Formatted using the formatting rules.